Clarify how to configure Ruler to send alerts to Alertmanager (#3685)
grafana/mimir: docs/sources/mimir/operators-guide/architecture/components/ruler/index.md

After the alert has been active for the entire `for` duration, it enters the **FIRING** (`firing`) state.
The ruler then notifies Alertmanagers of any **FIRING** (`firing`) alerts.

Configure the addresses of Alertmanagers with the `-ruler.alertmanager-url` flag. This flag supports the DNS service discovery format.
For more information about DNS service discovery, refer to [Supported discovery modes]({{< relref "../../../configure/about-dns-service-discovery.md" >}}).

> If you're using [Mimir's Alertmanager]({{< relref "../alertmanager.md" >}}), the address should point to Alertmanager's API.

Reviewer: ```suggestion
If you're using [Mimir's Alertmanager]({{< relref "../alertmanager.md" >}}), point the address to Alertmanager's API.
```

> Alertmanager's API prefix is configured by the `-http.alertmanager-http-prefix` flag, and defaults to `/alertmanager`.

Reviewer: ```suggestion
The Alertmanager's API prefix is configured via the `-http.alertmanager-http-prefix` flag, which defaults to `/alertmanager`.
```
